test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/CryptoTransferSuite.java

```diff
@@ -111,42 +111,52 @@ private HapiApiSpec tokenTransferFeesScaleAsExpected() {
 						tokenAssociate("f", "A", "B", "C"),
 						cryptoTransfer(tinyBarsFromTo("a", "b", 1))
 								.via("pureCrypto")
+								.fee(ONE_HUNDRED_HBARS)
```

Comment:
nit : Do we know why the tests are failing without specifying fee?

Reply:
I agree. We should not set explicit fee in `CryptoTransferSuite` unless it is intentional (usually when expecting `INSUFFICIENT_TX_FEE`).

Reply:
Yeah, to some extent. The fee estimation from client side is smaller as it doesn't take into consideration of custom fee, but still not clear why them are not the same for existing test cases without custom fee. That's why we're using the current implementation as noted. Need more dig to improve this.